apps/studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -122,10 +128,10 @@ const TableEditorMenu = () => {
         <div className="flex flex-col gap-1">
           <SchemaSelector
             className="mx-4 h-7"
-            selectedSchemaName={snap.selectedSchemaName}
+            selectedSchemaName={selectedSchema}
             onSelectSchema={(name: string) => {
               setSearchText('')
-              snap.setSelectedSchemaName(name)
```

Comment:
i haven't tested this locally but be careful with this - there are a few other components using snap.selectedSchemaName

if we're removing updating the schema name in the valtio store i think it'll break something
e.g ForeignKeySelector.tsx L301
e.g TableGridEditor L325 etc etc

Reply:
yup can confirm this breaks a few stuff - TableEditor L181
if you try go to another schema, create a table there, it will create the table in the public schema instead (since its the default on pg-meta's end)